lib/broadway_dashboard.ex

```diff
@@ -59,6 +59,7 @@ defmodule BroadwayDashboard do
         node = socket.assigns.page.node
 
         with :ok <- check_broadway_version(node),
+             :ok <- check_pipeline_running(node, pipeline),
```

Comment:
Do you really need this call? `Metrics.listen` is already going to check if it is running for you or not, right? Maybe have it return `{:error, :pipeline_not_found}` instead?

Reply:
It was not checking before. Now I moved the code to `Metrics.listen` :)